vignettes/prudence.Rmd

```diff
@@ -252,43 +287,20 @@ flights_frugal |>
   class()
 ```
 
-### Comparison
+### Comparison with similar tools
 
 Frugal duckplyr frames behave like lazy tables in dbplyr and lazy frames in dtplyr: the computation only starts when you *explicitly* request it with `collect.duckplyr_df()` or through other means.
 However, frugal duckplyr frames can be converted to lavish ones at any time, and vice versa.
 In dtplyr and dbplyr, there are no lavish frames: collection always needs to be explicit.
 
+## Conclusion
```

Comment:
I generally feel conclusions are not needed but in such a concept-heavy vignette it can be useful.

Reply:
and it gives a better home to the paragraph with further reading resources.